<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BUREvaluate</class>
 <widget class="QWidget" name="BUREvaluate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1007</width>
    <height>729</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BUR辊形评价</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="CardWidget" name="CardWidget_7">
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="2,13">
      <property name="spacing">
       <number>7</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="CardWidget" name="CardWidget_8">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>138</height>
         </size>
        </property>
        <layout class="QGridLayout" name="gridLayout_4" columnstretch="1,3">
         <item row="0" column="0">
          <widget class="CaptionLabel" name="CaptionLabel_5">
           <property name="font">
            <font>
             <pointsize>9</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>开始日期</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="CalendarPicker" name="CalendarPicker_3"/>
         </item>
         <item row="1" column="0">
          <widget class="CaptionLabel" name="CaptionLabel_6">
           <property name="font">
            <font>
             <pointsize>9</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>截至日期</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="CalendarPicker" name="CalendarPicker_4"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="CardWidget" name="CardWidget_9">
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="1" column="2">
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>132</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="1">
          <widget class="PrimaryPushButton" name="importPushButtonBUR">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>导入</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="3">
          <widget class="TableView" name="preViewTableBUR"/>
         </item>
         <item row="1" column="0">
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>133</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="CardWidget" name="CardWidget_10">
     <layout class="QVBoxLayout" name="verticalLayout_3" stretch="2,13">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="CardWidget" name="CardWidget_11">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>138</height>
         </size>
        </property>
        <layout class="QGridLayout" name="gridLayout_8">
         <item row="0" column="0" colspan="3">
          <widget class="SubtitleLabel" name="SubtitleLabel_2">
           <property name="text">
            <string>视为打满的百分比</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="ComboBox" name="ComboBoxBUR">
           <property name="text">
            <string>选择机架</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="4">
          <widget class="CaptionLabel" name="CaptionLabel_7">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>9</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>各个设定值占打满值的百分比，其中中间辊弯辊[0,2600]，中间辊窜辊[-142.5,142.5]，工作辊弯辊[-700,1000]</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="LineEdit" name="LineEditBUR">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="CaptionLabel" name="CaptionLabel_8">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>12</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>%</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2" colspan="2">
          <widget class="Slider" name="SliderBUR">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="2,1,0">
        <item>
         <widget class="CardWidget" name="CardWidget">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="TableView" name="highValueTableBUR"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="CardWidget" name="CardWidget_12">
          <layout class="QGridLayout" name="gridLayout_9">
           <item row="1" column="0" colspan="2">
            <widget class="CaptionLabel" name="SpecificLabelBUR">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="SubtitleLabel" name="SubtitleLabel_3">
             <property name="text">
              <string>3个设定值都打满卷数百分比</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="SubtitleLabel" name="PercentLabelBUR">
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer_8">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>99</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="PrimaryPushButton" name="displayPushButtonBUR">
            <property name="text">
             <string>打满显示</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="PrimaryPushButton" name="moreInfoPushButtonBUR">
            <property name="text">
             <string>详细信息</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_7">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>99</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ComboBox</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PrimaryPushButton</class>
   <extends>PushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>Slider</class>
   <extends>QSlider</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>CardWidget</class>
   <extends>QFrame</extends>
   <header>qfluentwidgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CalendarPicker</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>CaptionLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>SubtitleLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>TableView</class>
   <extends>QTableView</extends>
   <header>qfluentwidgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
